pkg/apis/proxy/v1alpha1/managedproxyconfiguration_types.go

```diff
@@ -204,6 +204,10 @@ type ManagedProxyConfigurationProxyServer struct {
 	// `entrypoint` defines how will the proxy agents connecting the servers.
 	// +optional
 	Entrypoint *ManagedProxyConfigurationProxyServerEntrypoint `json:"entrypoint"`
+
+	// `additionalArgs` defines args used in proxy-server.
```

Comment:
// `additionalArgs` adds arbitrary additional command line args to the proxy-server.

Reply:
Changed the description.